airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/e87ffa8e-a3b5-f69c-9076-6011339de1f6.json

```diff
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "e87ffa8e-a3b5-f69c-9076-6011339de1f6",
   "name": "Redshift",
   "dockerRepository": "airbyte/source-redshift",
-  "dockerImageTag": "0.3.7",
+  "dockerImageTag": "0.3.8",
```

Comment:
By the way, we no longer depend on these individual connector definitions. You don't need to update them any more. These files are still in our codebase for backward compatibility.

Actually I will check if we can remove all of them, since we already had that pseudo major version bump, and backward compatibility may not be needed.